pkg/controller/jobs/pod/pod_webhook.go

```diff
@@ -49,7 +49,7 @@ const (
 	ManagedLabelKey              = constants.ManagedByKueueLabel
 	ManagedLabelValue            = "true"
 	PodFinalizer                 = ManagedLabelKey
-	SuspendedByParentLabelKey    = "kueue.x-k8s.io/pod-suspending-parent"
+	SuspendedByParentAnnotation  = "kueue.x-k8s.io/pod-suspending-parent"
```

Comment:
```suggestion
	SuspendedByParentAnnotationKey  = "kueue.x-k8s.io/pod-suspending-parent"
```

Reply:
hm, but other annotation keys don't have the key suffix. So, I propose not to add it for consistency.

Reply:
but let me move down to be grouped with other annotations

Reply:
moved down to group with other annotations, but I will wait to confirm with adding the Key suffix